app/javascript/packs/restaurants.jsx

```diff
@@ -17,15 +24,143 @@ class Restaurants extends React.Component {
   }
 
   loadRestaurants = () => {
-    $.getJSON('/restaurants', (restaurants) => {
+    this.setState({ loadingRestaurants: true, showFilters: false });
+
+    const {
+      filterPickup: pickup,
+      filterDelivery: delivery,
+      filterUberEats: uber_eats,
+      filterSkipTheDishes: skip_the_dishes,
+      filterFoodora: foodora,
+      filterDoorDash: door_dash,
+    } = this.state;
+
+    const data = {
+      pickup,
+      delivery,
+      uber_eats,
+      skip_the_dishes,
+      foodora,
+      door_dash,
+    }
+
+    $.getJSON('/restaurants', data, (restaurants) => {
       this.setState({restaurants, loadingRestaurants: false});
     });
   }
 
+  toggleFilters = (e) => {
+    e.preventDefault();
+
+    const { showFilters } = this.state;
```

Comment:
No suggestion, just curious how this `showFilters` state works and why it's needed.